frontends/election-manager/src/components/print_button.tsx

```diff
@@ -66,7 +72,35 @@ export function PrintButton({
       document.title = documentTitle;
     }
 
+    setRenderPrintTarget(false);
     afterPrint?.();
+  }, [
+    afterPrint,
+    afterPrintError,
+    copies,
+    isMounted,
+    makeCancelable,
+    printer,
+    sides,
+    title,
+  ]);
+
+  useEffect(() => {
+    async function printOnTargetRendered() {
+      await print();
+    }
+
+    if (renderPrintTarget) {
+      void printOnTargetRendered();
+    }
```

Comment:
Curious: Why did you have to wrap `print` in `printOnTargetRendered`? I imagine there's a reason why you couldn't just write:

```ts
if (renderPrintTarget) {
  print();
}
```